lib/streamlit/watcher/event_based_path_watcher.py

```diff
@@ -339,19 +342,25 @@ def handle_path_change_event(self, event: events.FileSystemEvent) -> None:
             _LOGGER.debug("Don't care about event type %s", event.event_type)
             return
 
-        changed_path = os.path.abspath(changed_path)
+        # Watchdog 5.X is supports Python >=3.9, so watchdog 4.X is used for Python 3.8.
```

Comment:
super-nit: `Watchdog 5.X is supported by Python >=3.9`

Reply:
I appreciate it, most of the time I'm the one nitpicking others haha!

I updated the comment :)